ci/credentials.yml.example

```diff
@@ -8,13 +10,25 @@ cg_provision_development_git_branch: master
 status_access_token: CHANGEME
 slack-webhook-url: CHANGEME
 
+# This should be something like PROD or DEV
+env: CHANGEME
+
+# This is where the environment is deployed (AWS govcloud)
 aws_access_key_id: CHANGEME
 aws_secret_access_key: CHANGEME
 aws_default_region: CHANGEME
 aws_s3_tfstate_bucket: CHANGEME
+aws_s3_cloudtrail_bucket: CHANGEME
 aws_az1: CHANGEME
 aws_az2: CHANGEME
 
+lets_encrypt_hosted_zone_staging: CHANGEME
```

Comment:
Very minor, but I don't really see how these examples help. When most of the values are CHANGEME, this file isn't much more than a list of the variables we use in the pipeline, which we can already get from the pipeline. I'm not asking for a change here, just wondering if there's value in maintaining these example files.

Reply:
I think the idea is that this is for when you don't have a pipeline to start with, like when you are bootstrapping everything.  I've also used this when grepping as a sign that "oh, this is in the pipeline, I'll go look there".